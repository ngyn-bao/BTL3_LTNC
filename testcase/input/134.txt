BEGIN
INSERT x number
BEGIN
INSERT y string
RPRINT
END
END