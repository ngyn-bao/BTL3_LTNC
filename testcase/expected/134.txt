success
success
y//2 x//1